safe-global/web-core
# fix: Use fixed color for mobile pairing icon

## What it solves

Resolves #934 

## How this PR fixes it

- Uses a fixed color (#0FDA6D) for the mobile pairing icon

## How to test it

1. Open the safe
2. Connect via mobile app
3. Observe the icon is visible in light and dark mode

## Screenshots
<img width="243" alt="Screenshot 2022-10-21 at 15 54 18" src="https://user-images.githubusercontent.com/5880855/197212588-1fa3ca08-33b0-4031-b225-4998faccb6b3.png">

<img width="246" alt="Screenshot 2022-10-21 at 15 54 10" src="https://user-images.githubusercontent.com/5880855/197212556-a47e8304-67da-4935-bb7c-4faa5e9fb601.png">
